fractal-theme/scss/src/_default.scss

```diff
@@ -2,8 +2,9 @@ html,
 body,
 .Prose,
 .Tree-item {
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Helvetica Neue", Arial, sans-serif,
-    "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+
+  /* font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Helvetica Neue", Arial, sans-serif,
```

Comment:
Why not just delete it? I still think this font-stack is superior for body text, but if you want to theme it otherwise, I think you should just delete it?

Reply:
I like the idea. Don’t want to delete it just yet. But on windows the text is not pretty - I could not live with it :)
Maybe we want to use this again when we theme it